DMD2_Timer.cpp

```diff
@@ -136,11 +140,11 @@ void BaseDMD::begin()
 {
   beginNoTimer();
   timer0_detachInterrupt();
-  
+
   register_running_dmd(this);
-  
+
   timer0_isr_init();
-  timer0_attachInterrupt(scan_running_dmds);
+  timer0_attachInterrupt(esp8266_ISR_wrapper);
```

Comment:
Will changing this here break AVR builds? Suggest #ifdef-ing one or the other.

Reply:
Ahh, my boneheaded omission, will get on that asap.

Reply:
Since that appears to be okay, I will now dismiss the review and merge.